modules/collabora_online_group/tests/src/Kernel/AccessTest.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\collabora_online_group\Kernel;
+
+use Drupal\group\PermissionScopeInterface;
+use Drupal\Tests\collabora_online\Traits\MediaCreationTrait;
+use Drupal\Tests\collabora_online_group\Traits\GroupRelationTrait;
+use Drupal\Tests\group\Kernel\GroupKernelTestBase;
+use Drupal\Tests\media\Traits\MediaTypeCreationTrait;
+use Drupal\Tests\user\Traits\UserCreationTrait;
+use Drupal\user\RoleInterface;
+
+/**
+ * Tests Collabora group access.
+ */
+class AccessTest extends GroupKernelTestBase {
+
+    use MediaTypeCreationTrait;
+    use UserCreationTrait;
+    use MediaCreationTrait;
+    use GroupRelationTrait;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected static $modules = [
+        'file',
+        'media',
+        'image',
+        'group',
+        'groupmedia',
+        'collabora_online',
+        'collabora_online_group',
+        'user'
+    ];
+
+    /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
```

Comment:
Btw I was wondering what we do about the user id = 1 problem.
And I see that the parent class already takes care of it. Good!